xml/docker_building_images.xml

```diff
@@ -73,21 +79,23 @@ xml:id="docker.building.images" xml:lang="en">
   </para>
 
   <para>
-   After the pre-built images are installed, you need to list them using
+   If you've installed the pre-built images using Zypper, you need to list them using
    <literal>sle2docker</literal> to get a proper image name:
   </para>
 
 <screen>sle2docker list</screen>
 
   <para>
-   Now you need to activate the pre-built images:
+   Furthermore, if Zypper was used for installation, you need to activate the pre-built images:
   </para>
 
 <screen>sle2docker activate <emphasis>PRE-BUILT_IMAGE_NAME</emphasis></screen>
 
   <para>
    After successful activation, <literal>sle2docker</literal> will display the
-   name of the &docker; image. You can customize the docker image as described in
+   name of the &docker; image installed via Zypper.</para>
+
```

Comment:
These paragraphs are missing structure, because first it is unclear why one needs to get the image name.

--------------------
After you have obtained the image, you need to activate it. proceed as follows:

1. Get the proper image name by running ...
2. Activate the image by using the image name from the previous step ...
3. Check if the image was successfully activated by running sle2docker
---------------

Reply:
fixed, see 421b412